app/helpers/status_updates_helper.rb

```diff
@@ -0,0 +1,7 @@
+module StatusUpdatesHelper
+  def links_to_status_updates
+    if params[:action]=="edit"
```

Comment:
I _think_ there's also a helper method called `controller_action`. It's a bit more semantical than querying the params hash.